### DISCUSSION TITLE
Quarto doesn't follow symlinks

### DISCUSSION THREAD

**Discussion by yoctobug** [quarto-dev/quarto-cli#1415] [Q&A] [answered]
When using `quarto render myproject`, Quarto correctly renders files within the directory hierarchy specified, including recursing into subdirectories. However, it does not follow symlinks (and yes, the links are valid, and are resolved by my OS and by my IDE). 

When previewing I get a complaint:

     WARNING: Unable to resolve link target: foo/bar/baz.qmd

where `bar` is a symlinked directory.

Is there some way to get Quarto to follow symlinks? Or is that just not possible?

**Comment by jjallaire** [comment, marked as answer]
Currently Quarto doesn't support following symlinks (we found that in some cases users would have self-referential symlinks that would essentially create an infinite loop). If the problem is merely that our file globbing doesn't pick up your files, you might be able to workaround this by providing an explicit render list: https://quarto.org/docs/projects/quarto-projects.html#render-targets

**Reply by yoctobug** [reply]
Understood. TYVM.
